pkgs/development/ocaml-modules/lem/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, ncurses
+, makeWrapper
+, ocamlbuild
+, findlib
+, ocaml
+, num
+, zarith
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lem";
+  version = "2022-12-10";
+
+  src = fetchFromGitHub {
+    owner = "rems-project";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ZQgcuIVRkJS0KtpzjbO4OPHGg6B0TadWA6XpRir30y8=";
+  };
+
+  minimalOCamlVersion = "4.07";
+
+  nativeBuildInputs = [ makeWrapper ocamlbuild findlib ocaml ];
+  buildInputs = [ ncurses num ];
```

Comment:
Is `ncurses` really needed?

Reply:
no, i dont know why i data that (or why i put `num` in there either)